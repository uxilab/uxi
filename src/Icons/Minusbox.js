import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Minusbox = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 20 20"
      width="24px"
      height="24px"
    >
      <g id="Group_558" data-name="Group 558" transform="translate(-316 -361)">
        <g
          id="Rectangle_1_copy_4"
          data-name="Rectangle 1 copy 4"
          transform="translate(316 361)"
          fill="transparent"
          stroke="currentColor"
          strokeWidth="1"
          opacity="0.6"
        >
          <rect width="20" height="20" stroke="none" />

          <rect x="0.5" y="0.5" width="19" height="19" />
        </g>

        <rect
          id="Rectangle_86"
          data-name="Rectangle 86"
          width="11.88"
          height="2.253"
          transform="translate(320 370)"
        />
      </g>
    </svg>
  </SvgIcon>
);

Minusbox.propTypes = {
  color: PropTypes.string,
};

Minusbox.defaultProps = {
  color: 'currentColor',
};

export default Minusbox;
