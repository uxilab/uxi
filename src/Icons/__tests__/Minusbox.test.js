import React from 'react';
import Minusbox from '../Minusbox';

describe('Icon : <Minusbox />', () => {
  it('match snapshot', () => {
    expect(shallow(<Minusbox />)).toMatchSnapshot();
  });
});

