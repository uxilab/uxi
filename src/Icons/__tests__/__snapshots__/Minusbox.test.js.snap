// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Minusbox /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 20 20"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Group 558"
      id="Group_558"
      transform="translate(-316 -361)"
    >
      <g
        data-name="Rectangle 1 copy 4"
        fill="transparent"
        id="Rectangle_1_copy_4"
        opacity="0.6"
        stroke="currentColor"
        strokeWidth="1"
        transform="translate(316 361)"
      >
        <rect
          height="20"
          stroke="none"
          width="20"
        />
        <rect
          height="19"
          width="19"
          x="0.5"
          y="0.5"
        />
      </g>
      <rect
        data-name="Rectangle 86"
        height="2.253"
        id="Rectangle_86"
        transform="translate(320 370)"
        width="11.88"
      />
    </g>
  </svg>
</SvgIcon>
`;
